components/post_markdown/post_markdown.jsx

```diff
@@ -50,11 +53,20 @@ export default class PostMarkdown extends React.PureComponent {
         const proxyImages = !this.props.post || !this.props.post.message_source || this.props.post.message === this.props.post.message_source;
         const channelNamesMap = this.props.post && this.props.post.props && this.props.post.props.channel_mentions;
 
+        let {message} = this.props;
+        const {post} = this.props;
+
+        this.props.pluginHooks.forEach((o) => {
+            if (o && o.hook) {
+                message = o.hook(post, message);
```

Comment:
I kept the `message` param in case hooks want to build upon changes made by other hooks. Of course any given hook could choose to totally ignore the changes made by other hooks and simply use the `post.message` field.